samples/snippets/src/main/java/com/example/logging/LogEntryWriteHttpRequest.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.logging;
+
+// [START logging_write_request_entry]
+import com.google.cloud.MonitoredResource;
+import com.google.cloud.logging.HttpRequest;
+import com.google.cloud.logging.LogEntry;
+import com.google.cloud.logging.Logging;
+import com.google.cloud.logging.LoggingOptions;
+import com.google.cloud.logging.Payload;
+import com.google.cloud.logging.Severity;
+import java.util.Collections;
+
+/** Write LogEntry with HTTP request using the Cloud Logging API. */
+public class LogEntryWriteHttpRequest {
+
+  public static void main(String[] args) throws Exception {
+    // TODO(developer): Replace these variables before running the sample.
+    String logName = "log-name"; // i.e "my-log"
+    String payLoad = "payload"; // i.e "Hello world!"
+    HttpRequest httpRequest = HttpRequest.newBuilder().build();
+    // i.e HttpRequest request = HttpRequest.newBuilder()
+    //     .setRequestUrl("www.example.com")
+    //     .setRequestMethod(HttpRequest.RequestMethod.GET)
+    //     .setStatus(200)
+    //     .build();
```

Comment:
Any particular reason to not just use this uncommented in the sample?

Reply:
I have updated the code and removed commented section associated with HttpRequest.